packages/core/parcel/src/cli.js

```diff
@@ -70,6 +70,13 @@ const commonOptions = {
 
 var hmrOptions = {
   '--no-hmr': 'disable hot module replacement',
+  '--hmr-port <port>': [
```

Comment:
Why do we need a separate port? With `parcel serve`, it will be the same as `--port` and with watch we can just use `--port`

Reply:
that way it's explicit and clear to the CLI user that the default port for the HMR server is 1234, the same as the dev Server and where they're coming from, i.e not hard coded. **also** so you can start the HMR server on a different port if you need to. I think it's better to keep the `--port` option only for the dev/web server.
from the output of $ `parcel help watch` and $ `parcel help serve`:
```
--hmr-port <port>    set the port to serve HMR websockets,
   defaults to random if not available (default: 1234)
```
from the output of $ `parcel help serve` : 
```
  -p, --port <port>    set the port to serve on. defaults to 1234 (default: 1234)
```
note the `(default: 1234)` is added by the commander package.